### PR TITLE
Docs for Croeseid Testnet (v0.8) upgrade

### PR DESCRIPTION
**Pull request by lezzokafka**

### CODE REVIEW

**Review by yihuang — APPROVED** [review]

**Comment by yihuang** [comment]
```diff
diff --git a/docs/getting-started/croeseid-testnet-nix.md b/docs/getting-started/croeseid-testnet-nix.md
index 47ef12e..df0659f 100644
--- a/docs/getting-started/croeseid-testnet-nix.md
+++ b/docs/getting-started/croeseid-testnet-nix.md
@@ -20,7 +20,7 @@ $ cachix use crypto-com
 ## Install chain utils for testnet

 ```shell
-$ nix-env -iA chain-utils-testnet -f https://github.com/crypto-com/chain-main/archive/f53418d87b8fede4fcee59078c996ee000eae4d2.tar.gz
+$ nix-env -iA chain-utils-testnet -f https://github.com/crypto-com/chain-main/archive/6c75214a0d4a017b957d5538eef86d125114bdc7.tar.gz
 ```

 After installation, you'll have these commands in your `PATH`:
@@ -70,7 +70,7 @@ One of the strengths of nix package manager is you can have multiple isolated in
 you want to manage both testnet and mainnet chains or different versions of them on a single machine.

 ```shell
-$ nix build -f https://github.com/crypto-com/chain-main/archive/f53418d87b8fede4fcee59078c996ee000eae4d2.tar.gz chain-utils-testnet -o testnet
+$ nix build -f https://github.com/crypto-com/chain-main/archive/6c75214a0d4a017b957d5538eef86d125114bdc7.tar.gz chain-utils-testnet -o testnet
 $ export CHAINHOME=$PWD/testnet_data
 $ MONIKER=testnode ./testnet/bin/init-node
 $ ./testnet/bin/chain-main start --home $CHAINHOME
```

Can you add this change too?

File: docs/getting-started/croeseid-testnet.md [hunk 1]
```diff
@@ -115,12 +115,12 @@ Follow the below optional steps to enable state-sync.
 - (Optional)For state-sync configuration, in `~/.chain-maind/config/config.toml`, please modify the configurations of [statesync] `enable`, `rpc_servers`, `trust_height` and `trust_hash` by:
 
   ```bash
-  $ LASTEST_HEIGHT=$(curl -s https://testnet-croeseid-1.crypto.com:26657/block | jq -r .result.block.header.height); \
+  $ LASTEST_HEIGHT=$(curl -s https://testnet-croeseid.crypto.com:26657/block | jq -r .result.block.header.height); \
```
Comment (yihuang) [inline docs/getting-started/croeseid-testnet.md:118]: We need the new chain to run for a while before we can have at least 1000 blocks.

Comment (lezzokafka) [reply docs/getting-started/croeseid-testnet.md:118]: Yes we do, no worries 😉

**Comment by lezzokafka** [comment]
> ```diff
> nix build -f https://github.com/crypto-com/chain-main/archive/52d537481caf591066a2426d6bd3ef6c6b97e0db.tar.gz chain-utils-testnet -o testnet
> ```

Added 👍

File: docs/getting-started/croeseid-testnet-nix.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ One of the strengths of nix package manager is you can have multiple isolated in
 you want to manage both testnet and mainnet chains or different versions of them on a single machine.
 
 ```shell
-$ nix build -f https://github.com/crypto-com/chain-main/archive/f53418d87b8fede4fcee59078c996ee000eae4d2.tar.gz chain-utils-testnet -o testnet
+$ nix build -f https://github.com/crypto-com/chain-main/archive/52d537481caf591066a2426d6bd3ef6c6b97e0db.tar.gz chain-utils-testnet -o testnet
 $ export CHAINHOME=$PWD/testnet_data
 $ MONIKER=testnode ./testnet/bin/init-node
 $ ./testnet/bin/chain-main start --home $CHAINHOME
```
Comment (yihuang) [inline docs/getting-started/croeseid-testnet-nix.md:76]:
```suggestion
$ ./testnet/bin/chain-maind start --home $CHAINHOME
```

**Pull request merged**